### ISSUE TITLE
Intestinal serosal plaques

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11651]
**Preferred term label:**
Intestinal serosal lesions that begin as macules and evolve into papules and plaques

**Synonyms**

**Definition (free text, please give PubMed ID)**
Seen on laparoscopy or laparotomy 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Kohlmeier–Degos disease https://link.springer.com/article/10.1186/s13023-022-02322-9

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal Adding new term: 
- ID: HP:6001544
- label: Small intestinal porcelain-white serosal plaque
- parents: [Abnormal small intestinal mucosa morphology (HP:0025129)](https://hpo.jax.org/app/browse/term/HP:0025129)
- pmid: PMID:35443671
- def: Yellow plaques, said to resemble porcelain and located in the small intestinal serosa.
- comment: The disease in the gut is characterized by a purely arteriopathic and extravascular sclerosing process confined to the subserosal fat. The small- and medium-sized blood vessels within the subserosal fat exhibit an expansile fibrosing neointimal arteriopathy eventuating in acellular collagen plugs.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11651
- synonyms:

**Comment by MickeySegal** [comment]
It is a bit confusing that the label refers to white and the def refers to yellow.  This appears to be a way of describing porcelain-white as being yellowish, but should be made more explicit.
